src/main/java/org/jenkinsci/test/acceptance/plugins/csp/ContentSecurityPolicyReport.java

```diff
@@ -14,7 +15,11 @@ public ContentSecurityPolicyReport(Jenkins context) {
 
     public List<String> getReport() {
         List<String> lines = new ArrayList<>();
-        WebElement table = find(By.className("bigtable"));
+        WebElement table = getElement(By.className("jenkins-table"));
+        if (table == null) {
```

Comment:
I've seen cases where we weren't able to successfully browse to the CSP Report page (for example, because the test ended while logged in as a user without admin privileges), so I'm worried this would give a false negative. Can we instead look for the text "No reports" and only return an empty list in that case? Otherwise I think we would be hitting the case where we lack admin privileges and should throw an exception in that case.